DnDCharacterSheet/Models/Attribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Enums;
+
+namespace Models
+{
```

Comment:
use file scoped namespaces, this is an unnecessary level of identation, you can configure this in visual studio settings